CCC-445 Add java_runtime parameter (#16)
QubitProducts/dataflow_launcher: README.md

### How to run it? ###
`python3 -m dataflowlauncher.launcher`

> Make sure you have `flow.conf` in the current directory. In addition, the script assumes that the current directory is the git root or one of its subdirectories.

Reviewer: Maybe, this would be a bit clearer?
```suggestion
Make sure you have `flow.conf` in the current directory. In addition, the script assumes that the current directory resides inside a git repository.
```

`--jar_file`: Path to the JAR file. Overrides -j. Default: NONE

> `--java_runtime`: Alternative java runtime: Default `/usr/bin/java`

Reviewer: ```suggestion
`--java_runtime`: Alternative java runtime. Default: `/usr/bin/java`
```
